clojure-mode.el

```diff
@@ -196,6 +196,15 @@ Out-of-the box `clojure-mode' understands lein, boot, gradle,
           (and (listp value)
                (cl-every 'stringp value))))
 
+(defcustom clojure-elided-prefixes-ns
+  '()
+  "A list of additional regexp prefixes that are elided when formulating ns."
```

Comment:
I think we can do better with the wording here, as I wouldn't have figured out what does this do without seeing the code change.

Reply:
E.g. I think it's better to think here of directory structure and just mention that `clojure-expected-ns` happens to use this.

Reply:
I've renamed the option `clojure-directory-prefixes` with suggested default.

Reply:
@bbatsov I've renamed and established the default as suggested - any outstanding concerns? The patch is working well for me at the moment.